modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/services/SeqexecWebClient.scala

```diff
@@ -72,6 +85,7 @@ object SeqexecWebClient {
   def log(record: LogRecord): Future[Unit] =
     Ajax.post(
       url = s"$baseUrl/log",
-      data = default.write(LogMessage.fromLogRecord(record))
+      responseType = "arraybuffer",
+      data = Pickle.intoBytes(LogMessage.fromLogRecord(record))
     ).map(_.responseText)
 }
```

Comment:
I guess that's possible, I think it is very small in this case especially because they are repeated parameters rather than logic and in some cases we don't have data, e.g. on the `stop` command